submarine-cloud-v2/controller.go

```diff
@@ -1271,16 +1269,7 @@ func (c *Controller) syncHandler(workqueueItem WorkQueueItem) error {
 			return err
 		}
 
-		// Port-forwarding
-		// TODO:
-		//   (1) multi-tenant port-forwarding
-		//   (2) Basic operations: on/off/modify (change port)
-		//   (3) in-cluster
-		if action == ADD {
-			if !c.incluster {
```

Comment:
Do you have tested the deletion of submarine CR? I guess that the deletion will have some bugs.

Reply:
Oh I found that there's something I didn't notice,  I'll fix ASAP.